preflight.sh

```diff
@@ -20,12 +20,13 @@ if [ -z $PLATFORM ]; then
     exit 1
 fi
 
-if [[ $PLATFORM == "gce" ]]; then
-    # Set up network for running on gce
-    echo "Setting up network for GCE"
-    bash rto_setup.sh
+# Check if sudo is available
```

Comment:
This is a great change. Now it looks like we no longer need to pass in a `PLATFORM=...` flag anymore into this script and would no longer need this intermediate script `preflight.sh`. Instead we could just directly call `rto_setup.sh` in our scripts. We should change all of the instances where we call `preflight.sh` to just call `rto_setup.sh` instead. This is non-blocking for this PR but should be done in a follow-up PR.

Reply:
Could be worth it to keep a `preflight.sh` actually in case we add other initial workload set ups.

Reply:
Yes, I don't want to change the structure too much. We can save this for now since there might be more differences between GCE and GKE.